### PR TITLE
[MIG][13.0] stock_inventory_cost_info: Migration to v13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT21681
Please @Tardo @ernestotejeda review please

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: stock_inventory_cost_info/models/stock_inventory.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    )
+    adjustment_cost = fields.Monetary(
+        string="Adjustment cost", compute="_compute_adjustment_cost", store=True
+    )
+
+    @api.depends("product_qty", "theoretical_qty", "inventory_id.state")
+    def _compute_adjustment_cost(self):
+        for record in self:
+            adjusted_qty = record.product_qty - record.theoretical_qty
+            adjustment_cost = adjusted_qty * record.product_id.standard_price
```
Comment (ernestotejeda) [inline stock_inventory_cost_info/models/stock_inventory.py]:
```suggestion
            adjustment_cost = record.difference_qty * record.product_id.standard_price
```
I think we could use the new field 'difference_qty': https://github.com/odoo/odoo/blob/cedafe00941b869d284cd511633d7da15868fc0a/addons/stock/models/stock_inventory.py#L311

**Review by ernestotejeda — APPROVED** [review]
Thanks @sergio-teruel  , just one suggestion

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@sergio-teruel Could you look at @ernestotejeda review?

**Comment by pedrobaeza** [comment]
Aborting, as there's still a pending comment.

**Comment by dreispt** [comment]
Reviewers, if there are blocking issues, please set your review status accordingly, or add a "Needs Fixing" tag. Thanks!

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please attend @ernestotejeda's comment

**Comment by sergio-teruel** [comment]
@ernestotejeda Changes done!!!

**Comment by rousseldenis** [comment]
@sergio-teruel Could you check pre-commit ?

**Comment by sergio-teruel** [comment]
@rousseldenis rebase done and fix pre-commit!!

File: stock_inventory_cost_info/models/stock_inventory.py [hunk 2]
```diff
@@ -17,6 +17,6 @@ class InventoryLine(models.Model):
     @api.depends("product_qty", "theoretical_qty", "inventory_id.state")
     def _compute_adjustment_cost(self):
         for record in self:
-            adjusted_qty = record.product_qty - record.theoretical_qty
-            adjustment_cost = adjusted_qty * record.product_id.standard_price
-            record.adjustment_cost = adjustment_cost
+            record.adjustment_cost = (
+                record.difference_qty * record.product_id.standard_price
```
Comment (rousseldenis) [inline stock_inventory_cost_info/models/stock_inventory.py:21]: @sergio-teruel The depends should reflect the fields your computation depends on.

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM. One change required on depends

**Comment by sergio-teruel** [comment]
Done!!

**Review by pedrobaeza — APPROVED** [review]
It seems correct now. @rousseldenis can you update your review?

File: stock_inventory_cost_info/models/stock_inventory.py [hunk 3]
```diff
@@ -14,9 +14,9 @@ class InventoryLine(models.Model):
...
 
-    @api.depends("product_qty", "theoretical_qty", "inventory_id.state")
+    @api.depends("difference_qty", "inventory_id.state")
     def _compute_adjustment_cost(self):
         for record in self:
-            adjusted_qty = record.product_qty - record.theoretical_qty
-            adjustment_cost = adjusted_qty * record.product_id.standard_price
-            record.adjustment_cost = adjustment_cost
+            record.adjustment_cost = (
+                record.difference_qty * record.product_id.standard_price
```
Comment (rousseldenis) [inline stock_inventory_cost_info/models/stock_inventory.py:21]: Oups missing product_id.standard_price

Comment (pedrobaeza) [reply stock_inventory_cost_info/models/stock_inventory.py:21]: Not really, as we don't want to recompute all the past inventories when the cost price is changed. We only want to recompute it when validating or creating it.

Comment (rousseldenis) [reply stock_inventory_cost_info/models/stock_inventory.py:21]: Ok, makes sense

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**